src/summarycode/summarizer.py

```diff
@@ -502,11 +503,29 @@ def package_summarizer(resource, children, keep_details=False):
         logger_debug('package_summarizer: for:', resource,
                      'packages are:', packs)
 
+    package_purls = []
+    for package in packages:        
+        if package.get("purl", False):
+            package_purls.append(package["purl"])
```

Comment:
We prefer to use single quotes everywhere unless for docstring... do you mind to update these to use single quote for strings?

Reply:
Sure, I'll add it.